src/loadgenerator/Dockerfile

```diff
@@ -12,23 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM python:3.10-slim as base
+FROM python:3.10-slim AS base
 
-RUN apt-get -qq update \
-    && apt-get install -y --no-install-recommends \
-        g++ \
-        python3-dev
+WORKDIR /usr/src/app/
 
-COPY requirements.txt .
+COPY ./src/loadgenerator/ ./
```

Comment:
This line doesn't work with the current `compose.yaml` file.

The following change is necessary:
```yaml
  loadgenerator:
    build:
      context: ./
      dockerfile: ./src/loadgenerator/Dockerfile
```

Reply:
Thanks, good catch! I guess I forgot to build this one before publishing 😓